src/app/components/transaction/token-transfer-icon.tsx

```diff
@@ -0,0 +1,28 @@
+import { StacksTx } from '@shared/models/transactions/stacks-transaction.model';
+
+import { useCurrentAccountStxAddressState } from '@app/store/accounts/blockchain/stacks/stacks-account.hooks';
+
+import { ArrowDownIcon } from '../icons/arrow-down-icon';
+import { ArrowUpIcon } from '../icons/arrow-up-icon';
+import { MicroblockIcon } from '../icons/microblock-icon';
+import { getColorFromTx } from './transaction-type-icon';
+
+export function TokenTransferIcon(props: { tx: StacksTx }) {
+  const { tx } = props;
+  const currentAccountStxAddress = useCurrentAccountStxAddressState();
+  const isSent = tx.sender_address === currentAccountStxAddress;
+
+  if ('is_unanchored' in tx && tx.is_unanchored)
+    return (
+      <MicroblockIcon
+        bg={getColorFromTx(tx)}
```

Comment:
I think we should migrate the colors from `getColorFromTx` to our tokens

Reply:
Not sure what you mean here, they are tokens? 🤔 Look in `semantic-tokens.ts` file here.

Reply:
Sorry thats my mistake ✋  I didn't see them there